src/App.css

```diff
@@ -1,4 +1,277 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Roboto+Slab:wght@300;500;700&display=swap');
+
 * {
   margin: 0;
   padding: 0;
 }
+
+:root {
+  --white: #fff;
+  --blue: #0290ff;
+  --body: #fafafa;
+  --border: #e8e8e8;
+  --purple: #307bbe;
+  --light-purple: #4386bf;
+}
+
+body {
+  background-color: var(--body);
+}
+
+.gaps {
+  grid-gap: 40px;
+  gap: 40px;
+}
+
+.navbar {
+  padding: 24px 4%;
+  background-color: var(--white);
+  border-bottom: 1px solid var(--border);
+  display: flex;
+  align-items: center;
+}
+
+.navbar h1 {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: 700;
+  color: var(--blue);
+}
+
+.navbar ul {
+  display: flex;
+  flex-grow: 1;
+  list-style-type: none;
+}
+
+.navbar li a {
+  text-decoration: none;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  color: #121212;
+  letter-spacing: 1.9px;
+}
+
+.icon-button {
+  cursor: pointer;
+  width: 2.813rem;
+  height: 2.813rem;
+  border-radius: 50%;
+  border: 1px solid var(--border);
+  background-color: #fff;
+}
+
+.material-icons {
+  color: var(--blue);
+}
+
+.main-container {
+  padding: 4%;
+}
+
+.books-container ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  grid-gap: 0.938rem;
+  gap: 0.938rem;
+}
+
+.books-container ul li {
+  display: flex;
+  justify-content: space-between;
+  background-color: var(--white);
+  border: 1px solid var(--border);
+  padding: 2rem;
+}
+
+.books-info {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
```

Comment:
Please kindly reduce the repeated use of `flex property` in your CSS code as requested by the previous reviewer, so that we follow the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):. 🤗🤝💪